src/main/java/org/spongepowered/api/entity/living/Sittable.java

```diff
@@ -42,5 +42,4 @@
      * @param sitting Whether this entity is sitting
      */
     void setSitting(boolean sitting);
-
```

Comment:
Please remove these changes, as mentioned in the [contribution guidelines](https://docs.spongepowered.org/en/latest/devs/guidelines/#code-style) they're supposed to be there:

> Place a blank line before the first member of a class, interface, enum, etc. (i.e. after `class Example {`) as well as after the last member